lib/omniauth/login_dot_gov/userinfo_request.rb

```diff
@@ -33,7 +33,8 @@ def userinfo_from_response(response)
           social_security_number: parsed_body['social_security_number'],
           address: parsed_body['address'],
           phone: parsed_body['phone'],
-          phone_verified: parsed_body['phone_verified']
+          phone_verified: parsed_body['phone_verified'],
+          verified_at: parsed_body['verified_at'],
```

Comment:
Since MultiJson won't autoconvert this value from a string, please add logic that will parse the `verified_at` property into a `Time` object.

Reply:
nice catch! This should convert to a Time object, and I've updated my test as well